client/src/pages/RegisterPage.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+import { Form, Button } from "react-bootstrap"
+import axios from "axios"
+
+export default function RegisterPage() {
+
+    const [userInput, setUserInput] = useState({
+        email: "",
+        password: "",
+        confirmPassword: ""
+    })
+
+    const handleChange = (e) => {
+        setUserInput({
+            ...userInput,
+            [e.target.name]: e.target.value
+        })
+    }
+
+    const handleSubmit = async (e) => {
+
+        e.preventDefault()
+
+        let JWT = await axios.post("/auth/signup", {
+            email: userInput.email,
+            password: userInput.password,
+            confirmPassword: userInput.confirmPassword
```

Comment:
since the keys match, I think you can just send `...post("/auth/signup", userInput)`. Not a big deal. up to you if you want to swap it